pkg/controller/constraint/constraint_controller.go

```diff
@@ -112,6 +137,16 @@ func (r *ReconcileConstraint) Reconcile(request reconcile.Request) (reconcile.Re
 		return reconcile.Result{}, err
 	}
 
+	constraintKindName := strings.Join([]string{instance.GetKind(), instance.GetName()}, "/")
```

Comment:
When I suggested string joining, I was forgetting that 
```
struct {
 Kind string
 Name string
}
```
would also work.

Either method is fine with me.